mockgcp/docs/ai/create-mockgcp.md

```diff
@@ -47,15 +47,15 @@ You will also need to register the service in mockgcp/register.go by adding an a
 
 ## Running the test and capturing the real GCP output
 
-Now you can run the test and capture the real GCP output (it will be written to `mockgcp/mock<service>/testdata/<resource>/crud/_http.log`
+Now you can run the test and capture the real GCP output (it will be written to `mock<service>/testdata/<resource>/crud/_http.log`)
 
-Run `dev/tasks/record-gcp mock<service>/testdata/<resource>/crud` - that should succeed and write the _http.log
+Run `dev/tasks/record-gcp mock<service>/testdata/<resource>/crud` - that should succeed and write the dev/tasks/record-gcp mock<service>/testdata/<resource>/crud/_http.log
```

Comment:
Nit: good idea on providing the path , but I think `dev/tasks/record-gcp` will confuse things